### ISSUE TITLE
Automated tests: OSX because of SSL

### ISSUE THREAD

**Issue by hmaarrfk** [scikit-image/scikit-image#3308]
See this build for example:
https://travis-ci.org/scikit-image/scikit-image/jobs/411036356#L4748

```
216 
217     >>> picture = novice.open('http://scikit-image.org/_static/img/logo.png')
UNEXPECTED EXCEPTION: URLError(SSLError(1, '[SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:719)'),)
```

I tried setting http to https but that didn't fix things. Anybody have a clue?

**Comment by jni** [comment]
Plus I thought that novice had been removed?

**Comment by hmaarrfk** [comment]
Version 0.16!!!

**Comment by hmaarrfk** [comment]
Fixed in #3320
